epoch/api/chain_ws_api.md

```diff
@@ -323,8 +315,8 @@ Event:
 
   | Name | Type | Description | Required |
   | ---- | ---- | ----------- | -------- |
-  | hash | string | Hash of the new block | Yes |
-  | height | number | Height of the new block | Yes |
+  | hash | string | Hash of the mined key block or added micro block | Yes |
```

Comment:
Maybe the base58check block hash shall have a distinct prefix for key or micro block? Assessment can be in future PR.

Reply:
Distinct prefix for key and micro block planned for the next release with the introduction of the new HTTP API.